docs/v3.x/getting-started/angular.md

```diff
@@ -68,7 +68,7 @@ fetch('http://localhost:1337/restaurants', {
   },
 })
   .then(response => response.json())
-  .then(data => console.log(data));
+  .then(data => console.error(data));
```

Comment:
You went a bit too far :) console.error is only in  necessary in the catch ;)

Reply:
My editor changed all of them...
I'll fix it

Reply:
Done!